test/built-ins/Number/prototype/toFixed/return-values.js

```diff
@@ -8,5 +8,4 @@ description: >
   Return regular string values
 ---*/
 
-// a bug in IE ? - Edge 14
-assert.sameValue((1.255).toFixed(2), "1.25");
+assert.sameValue((1.25499999999999989342).toFixed(2), "1.25");
```

Comment:
@leobalter , 1.255 === 1.25499999999999989342 .
In case some engine will use binary256 arithmetic  `1.255` _could_ be closer to "1.26" , and so the test will be incorrect.